.github/workflows/continuous-delivery.yaml

```diff
@@ -18,14 +18,11 @@ jobs:
           fetch-depth: 0
           persist-credentials: false
       - name: Setup Node.js
-        uses: actions/setup-node@v3.5.1
-      - name: Cache dependencies
```

Comment:
Changing the caching mechanism to the one build-in in `actions/setup-node` is not exactly related to the issue, but since I'm updating the node version anyway I thought it makes sense to update this as well.